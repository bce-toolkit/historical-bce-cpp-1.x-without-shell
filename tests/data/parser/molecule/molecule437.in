NH4Al(SO4)2.12H2O
