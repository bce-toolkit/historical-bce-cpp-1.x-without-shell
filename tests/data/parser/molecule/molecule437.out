Al 1
H 28
N 1
O 20
S 2
